integration-tests/integration/alerts/createAlert.cy.js

```diff
@@ -7,7 +7,7 @@ const NotFoundPage = require('../../pages/notFound')
 context('A user can add an alert', () => {
   beforeEach(() => {
     cy.task('resetAndStubTokenVerification')
-    cy.task('stubSignIn', { username: 'ITAG_USER', caseload: 'MDI', roles: ['ROLE_UPDATE_ALERT'] })
```

Comment:
Given there are no exemptions in the cypress config any more, the only way many endpoints can be accessed is now if the user has no caseloads